setup.py

```diff
@@ -1,23 +1,27 @@
 from setuptools import setup
 
 setup(name='autodiscern',
-      version='0.0.1',
+      version='0.0.2',
       description='',
       url='https://github.com/CMI-UZH/auto-discern',
-      packages=['autodiscern'],
+      packages=['autodiscern', 'autodiscern.experiments', 'autodiscern.predictors'],
+      package_data={'autodiscern': ['package_data/*']},
       python_requires='>3.5.0',
       install_requires=[
```

Comment:
I just noticed we are missing Pytorch (version 1.2) https://pytorch.org/ and Pytorch pretrained BERT https://pypi.org/project/pytorch-pretrained-bert